### PR TITLE
Make sure to handle patch option for collection setData.

### PR DESCRIPTION
**Pull request by Natim**
Refs #188

### CODE REVIEW

File: test/collection_test.js [hunk 1]
```diff
@@ -278,12 +278,32 @@ describe("Collection", () => {
...
+        requests.updateRequest,
+        "/buckets/blog/collections/posts",
+        {
+          data: { a: 1 },
+          permissions: undefined,
+        },
+        {
+          headers: { Foo: "Bar", Baz: "Qux" },
+          patch: true,
+          safe: false,
```
Comment (Natim) [inline test/collection_test.js]: I am not sure about safe: false and last_modiifed: undefined here.

**Comment by Natim** [comment]
Tests fails but Travis is happy. https://travis-ci.org/Kinto/kinto-http.js/jobs/226332789

File: src/collection.js [hunk 2]
```diff
@@ -156,6 +156,7 @@ export default class Collection {
       { data, permissions },
       {
         last_modified,
+        patch: !!patch,
```
Comment (leplatrem) [inline src/collection.js]: I know that I'm the one who suggested you to put `patch: false` when unspecified, but I see that is not the way it's done elsewhere :/ eg. https://github.com/Kinto/kinto-http.js/blob/master/src/bucket.js#L150

File: README.md [hunk 3]
```diff
@@ -1594,3 +1595,8 @@ It's possible to run the integration test suite against an external Kinto server
 ```
 $ TEST_KINTO_SERVER=https://my.kinto-server.tld/v1 npm test
 ```
+
+### Releasing
+
+If you need to release a new version of kinto-http.js, you can follow
+[the Release documentation](https://github.com/Kinto/kinto-http.js/wiki/Releasing).
```
Comment (glasserc) [inline README.md]: 👍

**Comment by leplatrem** [comment]
> This was probably my fault, sorry!

I wouldn't say so... if there would have been tests for `collection.setData()` you would have spotted it ;)

**Pull request merged**